recipe/conda_build_config.yaml

```diff
@@ -224,8 +224,6 @@ pin_run_as_build:
     max_pin: x.x
   libmatio:
     max_pin: x.x
-  libnetcdf:
-    max_pin: x.x.x
```

Comment:
Was there a discussion that it's no longer necessary to pin libnetcdf build as run?  I would have thought this needed to be at least `x.x`.  That was my understanding of this comment: https://github.com/conda-forge/libnetcdf-feedstock/pull/90#issuecomment-558239517

Reply:
Maybe it's fine.  https://abi-laboratory.pro/index.php?view=timeline&l=netcdf shows 100% backwards compatibility for all the releases we're likely to have on conda-forge.

Reply:
@xylar with `run_exports` on the feedstock we don't need this here but the same pin still applies.